docs/adr/0017-switch-settings-auth-to-sessiontoken.md

```diff
@@ -0,0 +1,76 @@
+# Switch from OAuth2 to sharing sessionToken with GraphQL server for settings redesign auth
+
+- Status: proposed
+- Deciders: Les Orchard, Ben Bangert, Danny Coates, Ryan Kelly
+- Date: 2020-05-12
+
+Issue:
+
+- https://github.com/mozilla/fxa/issues/4850
+- https://jira.mozilla.com/browse/FXA-1559
+
+## Context and Problem Statement
+
+The Settings Redesign app needs to query & mutate the same protected user data as the existing settings app hosted on content-server. This will require some form of authentication & authorization to manage that data.
+
+In [ADR-0014](0014-auth-for-settings-redesign.md), we'd decided to use OAuth2 as the mechanism. But, upon attempting to implement, we discovered that [context missing from OAuth2 tokens][missing-oauth2-context] rendered the option unusable.
+
+[missing-oauth2-context]: https://github.com/mozilla/fxa/pull/4931#discussion_r411828476
+
+## Decision Drivers
+
+Basically, same as [ADR-0014](0014-auth-for-settings-redesign.md):
+
+- Smooth UX
+- Security
+- Development velocity
+- Ease of integration
+
+## Considered Options
+
+- Reuse existing sessionToken on content-server with auth-server APIs
+- Authenticate via OAuth2 to use auth-server APIs
+
+## Decision Outcome
+
+Chosen option: "Reuse existing sessionToken on content-server with auth-server APIs", because it's the least novel option requiring fewest changes to auth-server.
+
+### Positive Consequences
+
+- Easiest path to authenticating on auth-server for API requests, works almost exactly like the settings client does already.
+
+### Negative Consequences
+
+- We defeat the security promises of Hawk, but we're not really relying on them. We'd also like to move away from Hawk in the future anyway.
```

Comment:
Read without the benefit of several IRL discussions, I don't think it's clear how "Reuse existing sessionToken on content-server with auth-server APIs" defeats the security promises of hawk, or even what it's really proposing in concrete terms. It's probably worth a few more sentences here about the mechanics of what's being proposed, i.e. sending the sessionToken over the wire like a bearer token rather than using to to sign requests.